packages/core/src/components/TermList/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import withStyles, { WithStylesProps } from '../../composers/withStyles';
+import Term from './Term';
+
+export { Term };
+
+function TermList({
+  inline,
+  children,
+  cx,
+  styles,
+}: {
+  link?: NonNullable<React.ReactNode>;
+  endAlign?: boolean;
+  inline?: boolean;
+  keyTextRegular?: boolean;
+  children: NonNullable<React.ReactNode>;
+} & WithStylesProps) {
+  return <dl className={cx(inline && styles.inline, styles.termList)}>{children}</dl>;
+}
+
+export default withStyles(({ unit }) => ({
+  inline: {
+    display: 'inline-block',
+  },
+  termList: {
+    // Reset dl styles
+    marginBlockStart: 0,
+    marginBlockEnd: 0,
+    '@selectors': {
+      '> div:not(:last-child) > dd': {
```

Comment:
with the removal of the div in my other suggestion and with the goal to add spacing between `dt`s and `dd`s, this can be: 

```jsx
'> dd + dt': {
  marginTop: unit * 2,
},
```

or (wild guess)?

```jsx
'> dt:not(:first-child)': {
  marginTop: unit * 2,
},
```

Reply:
Went with `'> dd:not(:last-child)'` and it seems to be working